test/fixture/functions.output.json

```diff
@@ -52,22 +52,42 @@
     "line": 204
   },
   {
+    "message": "layers[12].paint.fill-antialias: property functions not supported",
+    "line": 217
+  },
+  {
     "message": "layers[12].paint.fill-antialias.stops[1]: object expected, number found",
     "line": 227
   },
   {
+    "message": "layers[13].paint.fill-antialias: property functions not supported",
+    "line": 241
+  },
+  {
     "message": "layers[13].paint.fill-antialias.stops[0]: object stop key must have zoom",
     "line": 244
   },
   {
+    "message": "layers[14].paint.fill-antialias: property functions not supported",
```

Comment:
It's probably unwise to have our style validator tests for property functions be based upon an unsupported property (`fill-antialias`). Could you switch these to another property?